pkg/trafficpolicy/types.go

```diff
@@ -62,9 +71,10 @@ type Rule struct {
 
 // OutboundTrafficPolicy is a struct that associates a list of Routes with outbound traffic on a set of Hostnames
 type OutboundTrafficPolicy struct {
-	Name      string                   `json:"name:omitempty"`
-	Hostnames []string                 `json:"hostnames"`
-	Routes    []*RouteWeightedClusters `json:"routes:omitempty"`
+	Name        string                   `json:"name:omitempty"`
+	Hostnames   []string                 `json:"hostnames"`
+	Routes      []*RouteWeightedClusters `json:"routes:omitempty"`
+	RetryPolicy RetryPolicy
```

Comment:
Since the Retry policy is associated with a route eventually, the RetryPolicy field should be a part of the RouteWeightedClusters struct, so that the code in envoy/rds/route can simply build the retry policy on a per route basis. Could you make this change to align with how this struct maps directly to envoy's route config?